.github/workflows/ci_cd.yml

```
@@ -822,13 +822,32 @@ jobs:
       - name: Run Playwright tests
         run: just frontend/run ${{ matrix.script }} --workers=2
 
+      - name: Check for changes after running tests with -u
+        id: check_snapshots_changed
```

It occurred to me when thinking about the condition on the step to upload the snapshot diff artefact, that we need _this_ step to run even if there testing step fails. By default, it wouldn't, so if there are failures in addition to changed snapshots, we'd miss the changed snapshots.

```suggestion
        id: check_snapshots_changed
        if: success() || failure()
```

---

@madewithkode it looks like this change got lost in the mix, but I think we still need it in the case I mentioned where there are multiple types of failures.